.storybook/preview.js

```diff
@@ -62,6 +78,28 @@ export const parameters = {
     },
 };
 
+const animationSpeed = [
+    0.1, 0.2, 0.5, 0.75, 1, 1.25, 1.5, 1.75, 2,
+    // Special value to make speed so fast that it could not be recognized
+    10000,
```

Comment:
```suggestion
    Infinity,
```

Reply:
We are calculating timing based on this number, we need some real number here

Reply:
`1 / Infinity === 0`

Reply:
But we don't need 0, we need some very small value to be sure that animation in JS will trigger

Reply:
we need it in storybook?

Reply:
I think it nice to have all options to see that components works properly with disabled animations

Reply:
and animation is disabled when duration is 0. I don't understand, how is this setting in storybook connected to testing animation?

Reply:
As i said before
> we need some very small value to be sure that animation in JS will trigger

Reply:
How it should look at the component level?

Reply:
Don't use `--yc-animation-multiplier` for critical animation, as was done in this PR for the `toaster` component. Or, for example, add small amount of time to result duration in critical animations.